propagator/ottrace/README.md

```diff
@@ -0,0 +1,43 @@
+# opentelemetry-propagator-ottrace
+
+The `opentelemetry-propagator-ottrace` gem contains injectors and extractors for the
+[ottrace context propagation format][ottrace-spec].
+
+## What is OpenTelemetry?
+
+[OpenTelemetry][opentelemetry-home] is an open source observability framework, providing a general-purpose API, SDK, and related tools required for the instrumentation of cloud-native software, frameworks, and libraries.
+
+OpenTelemetry provides a single set of APIs, libraries, agents, and collector services to capture distributed traces and metrics from your application. You can analyze them using Prometheus, Jaeger, and other observability tools.
+
+## How does this gem fit in?
+
+This gem can be used with any OpenTelemetry SDK implementation. This can be the official `opentelemetry-sdk` gem or any other concrete implementation.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-propagator-ottrace
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-propagator-ottrace` in your `Gemfile`.
```

Comment:
Absolutely. I will it as a line item in the TODO section.

Reply:
just a note that it will be good to mention the env var setting for this that's wired up `OTEL_PROPAGATORS=ottrace` https://github.com/open-telemetry/opentelemetry-ruby/blob/be5e882815c4d0e595d9fa1e94f764bf28c3004c/sdk/lib/opentelemetry/sdk/configurator.rb#L175